packages/strapi-design-system/src/DatePicker/__tests__/DatePicker.spec.tsx

```diff
@@ -394,6 +394,48 @@ describe('DatePicker', () => {
 
       expect(getByRole('gridcell', { name: '2022年12月18日日曜日' })).toHaveAttribute('aria-selected', 'true');
     });
+
+    /**
+     * TODO: the spying doesn't work :(
+     */
+    it.skip('should not change the date automatically if the timezone is not UTC but is America/Los_Angeles', async () => {
```

Comment:
Is this test meant to be skipped? it seems to be working for me

Reply:
It still passes because the timezone is set to UTC by default inside the component.